lib/crypto/crypto_scrypt-ref.c

```diff
@@ -26,7 +26,7 @@
  * This file was originally written by Colin Percival as part of the Tarsnap
  * online backup system.
  */
-#include "scrypt_platform.h"
+#include "platform.h"
```

Comment:
I just noticed that this line can be deleted entirely -- there's no system-specific `#ifdef`s here.  (at least none that use `config.h`)